lib/faerie/src/backend.rs

```diff
@@ -31,14 +33,16 @@ impl FaerieBuilder {
         isa: Box<TargetIsa>,
         name: String,
         format: container::Format,
-    ) -> Result<Self, Error> {
+        collect_traps: bool,
```

Comment:
Could you add some documentation mentioning what the traps argument does? Just clarify that this enables the `trap_manifest` member from the `FaerieProduct`.